test/test/util/manifest_generator.js

```diff
@@ -353,7 +365,8 @@ shaka.test.ManifestGenerator.prototype.addAudio = function(id) {
   }
 
   if (!stream)
-    stream = this.createStream_(id, ContentType.AUDIO, variant.language);
+    stream = this.createStream_(id, ContentType.AUDIO, variant.language,
+        variant.audio.label);
```

Comment:
This looks like it will throw an exception, since variant.audio doesn't exist yet.